lib/control-plane/kustomization.yaml

```diff
@@ -10,6 +10,19 @@ secretGenerator:
     options:
       disableNameSuffixHash: true
 
+  - name: libvirt-secret
```

Comment:
Can this not be in the generic control plane that all CRs use?

Reply:
If I understand this PR https://github.com/openstack-k8s-operators/dataplane-operator/pull/831 correctly, it is required by all deployments now.

Reply:
Resolved by https://github.com/openstack-k8s-operators/architecture/pull/188 which is merged.

I see that this has the following, which will blend fine with it when this  is rebased.

https://github.com/openstack-k8s-operators/architecture/compare/a06a444e7aa2cb1f80aaaf0a08a5d61d0bf38dc3..ac8bb7d7456a1fc4b3d4eefbf4e980d3ee93391a